bitbots_quintic_walk/include/bitbots_quintic_walk/walk_node.h

```diff
@@ -5,6 +5,7 @@ The original files can be found at:
 */
 #ifndef BITBOTS_QUINTIC_WALK_INCLUDE_BITBOTS_QUINTIC_WALK_WALK_NODE_H_
 #define BITBOTS_QUINTIC_WALK_INCLUDE_BITBOTS_QUINTIC_WALK_WALK_NODE_H_
+#define M_TAU M_PI * 2
```

Comment:
I think it is better style to define a constant like this in the cpp file in case that this header file is included elsewhere and messes with other people's code. (Same for the other `M_TAU` definitions below.)